packages/destination-actions/src/destinations/talon-one/createAudience/index.ts

```diff
@@ -0,0 +1,34 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Create Audience',
+  description: 'Create Audience in Talon.One',
+  fields: {
+    audience_id: {
+      label: 'audience_id',
+      description: 'Segment Audience ID (Third party audience id)',
+      type: 'string',
+      required: true
+    },
+    audience_name: {
+      label: 'audience_name',
+      description: 'Segment Audience Name (Third party audience name)',
+      type: 'string',
+      required: true
```

Comment:
@bolatkhanpm create audience

Reply:
You should get this Audience ID from Segment.

Reply:
This creates a new audience entity in Talon.One.